drivers/vulkan/rendering_device_vulkan.cpp

```diff
@@ -3364,8 +3364,14 @@ VkRenderPass RenderingDeviceVulkan::_render_pass_create(const Vector<AttachmentF
 	attachment_last_pass.resize(p_attachments.size());
 
 	Vector<VkAttachmentDescription> attachments;
+	Vector<int> attachment_remap;
 
 	for (int i = 0; i < p_attachments.size(); i++) {
+		if (p_attachments[i].usage_flags == 0xFFFFFFFF) {
```

Comment:
I would add a enum { UNUSED_ATTACHMENT = 0xFFFFFFFF }; to RenderingDevice::AttachmentFormat